pyro/poutine/trace_poutine.py

```diff
@@ -41,16 +41,24 @@ class TraceMessenger(Messenger):
     We can also use this for visualization.
     """
 
-    def __init__(self, graph_type=None):
+    def __init__(self, graph_type=None, param_only=None):
         """
         :param string graph_type: string that specifies the type of graph
```

Comment:
add `param_only` to docstring

Reply:
Also, `__init__` methods typically do not have docstrings, instead this information should be at the end of the class docstring (unrelated to this PR, but might as well change since you're editing the docstring).

Reply:
oh right i didnt even notice that